p2p/common/protocol.go

```diff
@@ -61,6 +65,10 @@ func MessageTypeFromString(mt string) MessageType {
 		found = MessageTypeSendAnchoredDoc
 	} else if MessageTypeSendAnchoredDocRep.Equals(mt) {
 		found = MessageTypeSendAnchoredDocRep
+	} else if MessageTypeGetDoc.Equals(mt) {
+		found = MessageTypeGetDoc
+	} else if MessageTypeGetDocRep.Equals(mt) {
+		found = MessageTypeGetDocRep
```

Comment:
@charleenfei could you change this entire block to use a switch statement instead?

Reply:
will do!